features/cvmfs/rpms.pan

```diff
@@ -5,10 +5,10 @@ unique template features/cvmfs/rpms;
 include 'quattor/functions/package';
 
 '/software/packages' = {
-  if ((pkg_compare_version(CVMFS_CLIENT_VERSION, '2.1.20') >= 0) && (OS_VERSION_PARAMS['major'] != 'sl5')) {
-    SELF[escape('cvmfs-config-default')] = dict();
-  } else {
+  if ((pkg_compare_version(CVMFS_CLIENT_VERSION, '2.1.20') == PKG_VERSION_LESS) || (OS_VERSION_PARAMS['major'] == 'sl5')) {
```

Comment:
Yes it is (it should)! I inverted the test due to the problem described in https://github.com/quattor/template-library-core/issues/132. Before the test was `pkg_compare_version <= 0` which meant (if I am right) greater or equal. I swaped the if conditions and thus test for `less`.